src/node/nodestate.h

```diff
@@ -1162,6 +1096,121 @@ namespace ccf
       secrets_view->put(version, secret_set);
     }
 
+    KeyShareInfo generate_key_share_info(
+      const std::vector<ccf::MemberPubInfo>& members_info)
+    {
+      auto share_wrapping_key_raw =
```

Comment:
Once we've got https://github.com/microsoft/CCF/issues/793, we should clear this variable once it has been split.